widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css

```diff
@@ -4852,6 +4852,11 @@ div.listbox .pentaho-listbox {
   border: 1px solid #7EC9F2;
 }
 
+div.gem:hover[type="measure"] {
+  background: #DDF4FC;
+  border: 1px solid #7EC9F2;
+}
+
```

Comment:
1. Shouldn't this use the class `.over`, instead of the pseudo-class `:hover`?
2. Shouldn't the over style use different colors than the normal style?